recipes/tl-expected/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
+
+# CTest is a testing tool that can be used to test your project.
```

Comment:
please, remove the commented code.

Reply:
Commented code removed in 4baaac1af3280c4e2d7eeea68e807f83e9d72707.